mappings/net/minecraft/client/render/item/BannerRenderers.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS net/minecraft/class_5876 net/minecraft/client/render/item/BannerRenderers
```

Comment:
`BannerRenderer`*?

Reply:
The point of this class is to provide 2 fields implementing `BannerRenderers.Renderer`

Reply:
I think there are still better names for it, but I guess we can leave that up in the air for now.